doc/source/ray-core/doc_code/task_exceptions.py

```diff
@@ -35,4 +35,47 @@ def g(x):
     # Exception: the real error
 
 # __task_exceptions_end__
+# __catch_user_exceptions_begin__
+
+class MyException(Exception):
+    ...
+
+@ray.remote
+def raises_my_exc():
+    raise MyException("a user exception")
+try:
+    ray.get(raises_my_exc.remote())
+except MyException as e:
+    print(e)
+    # ray::raises_my_exc() (pid=15329, ip=127.0.0.1)
+    #   File "/Users/ruiyangwang/gits/ray/doc/source/ray-core/doc_code/task_exceptions.py", line 45, in raises_my_exc
+    #     raise MyException("a user exception")
+    # MyException: a user exception
+
+# __catch_user_exceptions_end__
+# __catch_user_final_exceptions_begin__
+
+from typing import final
+
+@final
+class MyFinalException(Exception):
+    ...
```

Comment:
As we know now that we can still subclass this so this doc is no longer accurate.